### DISCUSSION TITLE
Im not sure how to get this on mac os.

### DISCUSSION THREAD

**Discussion by woodenlalaloo** [SteamRE/DepotDownloader#323] [Q&A] [answered]
I'm running mac os montrey on an m1 iMac but downloading the latest release and unzipping just shows me a few files that look like config files and an exe. Not sure how to run an exe on mac.

**Comment by psychonic** [comment, marked as answer]
It's similar to running any other .NET console app. Download and install the .NET 6 runtime if you don't have it already, and then:
`dotnet DepotDownloader.dll <args>`

**Reply by woodenlalaloo** [reply]
ty  i will try
